rust/processor/src/db/parquet/models/default_models/parquet_block_metadata_transactions.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use crate::{
+    bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable},
+    db::common::models::default_models::raw_block_metadata_transactions::{
+        BlockMetadataTransactionConvertible, RawBlockMetadataTransaction,
+    },
+};
+use allocative_derive::Allocative;
+use field_count::FieldCount;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Allocative, Clone, Debug, Default, Deserialize, FieldCount, Serialize, ParquetRecordWriter,
+)]
+pub struct BlockMetadataTransaction {
+    pub txn_version: i64,
+    pub block_height: i64,
+    pub id: String,
```

Comment:
any clue what id here represents?

Reply:
synced offline, we will rename this to block_id